lib/find_retryables.ts

```diff
@@ -190,31 +173,41 @@ const main = async (
     let isContinuous = options.continuous
     let fromBlock = options.fromBlock
     let toBlock = options.toBlock
+
     while (isContinuous) {
       const lastBlockChecked = await checkRetryablesOneOff(fromBlock, toBlock)
-      // Update isContinuous based on dynamic condition (if needed)
       isContinuous = options.continuous
-      // start at last block checked
       fromBlock = lastBlockChecked
-      // set to block to 0 (will use latest block)
       toBlock = 0
-      // Add a delay between continuous checks (e.g., wait for 30 minutes)
-      await new Promise(resolve => setTimeout(resolve, 30 * 60 * 1000))
+      await new Promise(resolve => setTimeout(resolve, 30 * 60 * 1000)) // 30 minutes delay
     }
   }
 
-  // Run either a one-off check or continuous check based on the option
   if (options.continuous) {
     await checkRetryablesContinuous()
   } else {
     await checkRetryablesOneOff(options.fromBlock, options.toBlock)
   }
 }
 
-// Calling the main function with the provided options
-main(networkConfig, options)
-  .then(() => process.exit(0))
-  .catch(error => {
-    console.error(error)
-    process.exit(1)
-  })
+const configFileContent = fs.readFileSync(
+  path.join(process.cwd(), 'lib', options.configPath),
+  'utf-8'
+)
+
+const config = JSON.parse(configFileContent)
+
+if (!Array.isArray(config.childChains)) {
+  console.error('Error: Child chains not found in the config file.')
+  process.exit(1)
+}
+
+const processOrbitChainsConcurrently = async () => {
```

Comment:
The problem with showing the results concurrently is that it mixes the result of both chains, making the output hard to read. An alternative can be to store all results somewhere and then log the results of each chain independently.
Than can help you separate the logic of the function from the "rendering" of the results too.